zeppelin-zengine/src/main/java/org/apache/zeppelin/helium/HeliumBundleFactory.java

```diff
@@ -101,8 +101,8 @@ public HeliumBundleFactory(
     this.heliumBundleDirectory = new File(heliumLocalRepoDirectory, HELIUM_BUNDLES_DIR);
     this.heliumLocalModuleDirectory = new File(heliumLocalRepoDirectory, HELIUM_LOCAL_MODULE_DIR);
     this.yarnCacheDir = new File(heliumLocalRepoDirectory, YARN_CACHE_DIR);
-    this.conf = conf;
-    this.defaultNpmRegistryUrl = conf.getHeliumNpmRegistry();
+    this.defaultNpmRegistryUrl = conf.getHeliumNpmRegistryUrl();
```

Comment:
We need proxies for 

- node installation 
- npm installation (not npm packages, npm itself)
- yarn installation

https://github.com/eirslett/frontend-maven-plugin#proxy-settings